src/components/Home.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Grid } from '@material-ui/core';
+import HomeButtonsMajor from './HomeButtonsMajor';
+import HomeButtonsMinor from './HomeButtonsMinor';
+import Search from './Search';
+import { THomeButton } from '../types';
+
+interface Props {
+  classBlock: string;
```

Comment:
Would you mind putting a comment here about what these props mean/do (maybe an example). I don't really get it.

Reply:
These are just part of the [BEM class naming convention](http://getbem.com/).